addons/website/controllers/main.py

```diff
@@ -1046,6 +1046,37 @@ def actions_server(self, path_or_xml_id_or_id, **post):
 
         return request.redirect('/')
 
+    # Enable the footer template and its corresponding copyright template.
```

Comment:
This commit should be squashed with the one before: it's the previous commit that introduce the bug you are solving here, we should not have a commit fixed by another in the same PR (except on very large PR that would probably be merged with a merge commit and not rebase-ff).